config/mock_responses/adoption.json

```diff
@@ -0,0 +1,35 @@
+{
+  "results": [
+    {
+      "technology_name": "Wordpress",
+      "dates": [
+        {
+          "date": "2022-04-22T00:00:00.000Z",
+          "origins": {
+            "mobile": 524362,
+            "desktop": 73542
+          }
+        },
+        {
+          "date": "2022-03-22T00:00:00.000Z",
+          "origins": {
+            "mobile": 471531,
+            "desktop": 68634
+          }
+        }
+      ]
+    },
+    {
+      "technology_name": "Squarespace",
+      "dates": [
+        {
+          "date": "2022-04-22T00:00:00.000Z",
+          "origins": {
+            "mobile": 16345,
+            "desktop": 735
+          }
+        }
+      ]
+    }
+  ]
+}
```

Comment:
One minor nit. Do you have editorconfig set up on your IDE? We have a config file to auto add final line breaks (plus other things) to stop GitHub flagging these:

<img width="110" alt="image" src="https://github.com/HTTPArchive/httparchive.org/assets/10931297/8b4e331f-2396-46dc-9cb5-6cab873509b0">